concurrent/src/main/java/io/airlift/concurrent/VirtualThreads.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.airlift.concurrent;
+
+import java.util.concurrent.Executor;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ThreadFactory;
+
+// https://github.com/jetty/jetty.project/commit/f8601750f3e2e1ab8626e15f31d25034ebd3a0b9
+public class VirtualThreads
+{
+    private VirtualThreads() {}
+
+    public static Executor getNamedVirtualThreadsExecutor(String prefix)
```

Comment:
Maybe we should move to Java 21

Reply:
I'm fine with that. This class could be replaced with a Jetty-provided one, once 12.0.7 is out so I'll drop this commit eventually.